### ISSUE TITLE
Update RASTER_API examples for pgstac titler v0.8

### ISSUE THREAD

**Issue by anayeaye** [NASA-IMPACT/veda-docs#106]
# What
This is a list of notebooks and updates that will be needed when the upgraded titiler is released. I ran this work locally against https://test-raster.delta-backend.com/ some of the changes pre-date the most recent upgrade.
> ~This is blocked until the titiler upgrade is released for the staging API.~ 2023-11-16: the titiler upgrade is released

## PI Objective 
https://github.com/NASA-IMPACT/veda-architecture/issues/333

# Why not just a PR?
We check in evaluated notebooks so instead; let's reproduce these changes after the API is live.

# AC
Implement the following changes
- [x] `statistics/` results for bands as b1, b2, b3 instead of 1, 2, 3 `notebooks/datasets/ocean-npp-timeseries-analysis.ipynb`
In both notebooks the clean_stats method used to shorten the keys in the results of `f"{RASTER_API_URL}/cog/statistics"` need to be updated to use statistics.b1 instead of statistics.1. This comes from an earlier titiler change that normalized band keys in stats responses.
```
def clean_stats(stats_json) -> pd.DataFrame:
    df = pd.json_normalize(stats_json)
    df.columns = [col.replace("statistics.b1.", "") for col in df.columns]
```
- [x] `statistics/` exactly the same `clean_stats()` change as above notebooks/quickstarts/timeseries-stac-api.ipynb`
- [ ] `notebooks/tutorials/gif-generation.ipynb` use `cog/feature` instead of `cog/crop`. **Header text and code** changes needed
- [x] `helper_functions.py`  use `cog/feature` instead of `cog/crop`
- [x] `notebooks/quickstarts/hls-visualization.ipynb` has older changes unrelated to upgrade. First we need to correct the band expressions defined a the top of the notebook by adding `_b1` . Then we should add back the third tiling example so that we cover all three cases (Map 1 = shows how to apply post process algorithm, Map 2 = demonstrates composite of multiple bands, Map 3 (previously removed because of band expression error) shows applying NDVI band math in the tile url. 
  - [x] `s30_vegetation_index_expression = "(B08_b1-B04_b1)/(B08_b1+B04_b1)"`
  - [x] `l30_ndwi_expression = "(B03_b1-B05_b1)/(B03_b1+B05_b1)"`
  - [x] replace `"post_process": "swir"` with `"algorithm": "swir"` in example map one of SWIR post processing
  - [x] Return the last (L30 Hurricane Maria) example from the original notebook [here](https://github.com/NASA-IMPACT/veda-documentation/blob/main/hls-visualization.ipynb). It will run as written now that the expressions have been corrected (`_b1`)

**Comment by anayeaye** [comment]
Update: this is no longer blocked. The staging stack is now running with the upgraded titiler code.

**Comment by jsignell** [comment]
Is this what you would expect for the hurricane maria ndwi? Naively it looks like there is some kind of offset relative to the water features on the basemap

![image](https://github.com/NASA-IMPACT/veda-docs/assets/4806877/d2d19dc3-b62e-48dd-88ee-4ca48744a949)

**Comment by anayeaye** [comment]
@jsignell I see what you mean about the apparent offset and do not know if it should be expected for the index (I had selected one just to showcase the different options for visualizing multi band data. We have observed some differences in how tiles are rendered with the latest raster-api and it is possible this visualization could have changed.

@brianmfreitag does this look right to you and/or you recommend another index?
